src/element/shape.js

```diff
@@ -611,8 +611,8 @@ export class ShapeElement extends Element {
           if (!animData.fromPath) return false;
           const animatePath = ShapeElement.createAnimatePath(animData, frame);
           this.drawPath(animatePath);
-          if (index !== 0 && this.graphicsData.length > 1) {
-            this.addHole();
+          if (index !== 0) {
+            this.endHole();
```

Comment:
_"... In v5, Graphics has simplified and the API changed from addHole to beginHole and endHole."_
[Graphic holes](https://github.com/pixijs/pixi.js/wiki/v5-Migration-Guide#graphics-holes)